testplan/common/remote/remote_service.py

```diff
@@ -122,8 +146,8 @@ def starting(self):
             f"\tErr file: {self.std.err_path}\n"
         )
 
-    def _wait_started(self, timeout=None):
-
+    def _wait_started(self, timeout: float = None) -> None:
+        """TODO"""
```

Comment:
any more todo here?

Reply:
? There was no docstring originally, but for now, I have not gone through this completely and simply added a TODO.